source/_integrations/bt_smarthub.markdown

```diff
@@ -29,6 +29,11 @@ host:
   default: 192.168.1.254
   required: false
   type: string
+smarthub_model:
+  description: The model of your BT Smarthub 1 or 2. The default will attempt to detect the hub type.
+  default: null
```

Comment:
I think this is very confusing for our users. Is the default 1 or 2?

Reply:
The options are null, 1 and 2, with the default being null. This is a power user config option which allows you to set which model of the smart hub you have. The majority of our users should never need to set the `smarthub_model` config option as the component will try and detect the model if no conf is set.

But I appreciate that it is confusing and needs to be improved because I just wrote 3 sentences explaining it to you lol. I'm not sure what our convention is for explaining this kind of optional config.

Reply:
Well let's leave the default out of the documentation in that case 😉

Reply:
Since there is no default, it still should be removed.

```suggestion
```